Quantum-Control-Applications/Optically addressable spin qubits/NV center in a confocal setup/g2_MZI_experiment.py

```diff
@@ -0,0 +1,132 @@
+"""
+g2_MZI_experiment.py: Single NVs confocal microscopy - Intensity auto-correlation g2 using 2 channels in a Mach-Zender Interferometer.
+"""
```

Comment:
Was this tested in a lab? 
I think we only want to add tested stuff here, but I'm not sure.
@TheoLaudatQM ?